### ISSUE TITLE
Dative Pronouns

### ISSUE THREAD

**Issue by nirjan-dev** [nirjan-dev/learning-german#30]
# Personal Pronouns in the Dative Case
Many words change in the dative case. For the third person pronouns, the following are different from the nominative case: the masculine pronoun is ihm (to him), the feminine is ihr (to her), the neuter is ihm (to it), and the plural is ihnen (to them).

Nominative | Accusative | Dative
-- | -- | --
ich (I) | mich (me) | mir (to me)
du | dich | dir
er / es / sie | ihn / es / sie | ihm / ihm / ihr
wir | uns | uns
ihr | euch | euch
sie | sie | ihnen

Some observations:

   - In dative, mir, dir, ihr (to me / you / her) rhyme.

  -  In the third person, the endings are the same as for the articles: -m, -r, -n. However, plural dative is "ihnen" (not ihn, as you might expect).

   - In the second person plural, accusative and dative pronouns are the same.

Now you can understand why, when thanking a female person, it is only correct to say Ich danke ihr ("I thank her", literally "I give-thank to her") and not Ich danke sie (that sounds like "I thank she" would sound to an English speaker).

# dative verbs

Remember that some verbs have a dative object. This is just a quirk of German. There was a reason for it when these words were created, but it's not easy to understand anymore, after a lot of language change.

In short, you just have to learn these :) There aren't very many.

Gehören literally means to "belong to". But don't translate too literally, often a different translation will be more natural.

-    Wem gehört das Kleid? ("Whose dress is it?" - Literally, "Whom does the dress belong to?")
